docs/planning/frontend/src/pages/EvaluatePage.jsx

```
@@ -0,0 +1,250 @@
+          </div>
+          {reviewers.length === 0 && (
+            <p className="text-sm text-amber-600 mt-2">
+              No reviewers found. Go to Import → Seed Reviewers to create default family members.
+            </p>
+          )}
+        </div>
+
+        {/* Rating */}
+        <div>
+          <label className="label">Rating *</label>
+          <StarRating
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)